doorstop/core/tree.py

```diff
@@ -176,9 +176,13 @@ def _place(self, document):
             log.info("parent options: {}".format(prefixes))
             raise DoorstopError(msg)
 
+        self.regenerate_children()
+
+    def regenerate_children(self):
```

Comment:
In the future, you can just amend or revert commits in the same branch. I merged #251. Thanks!